### PR TITLE
Feature/opsexp 24 improve helm

### PR DESCRIPTION
**Pull request by sergiuv2020**
Added helm values for 6.2.N, 6.1.N, 6.0.N
Added docker-compose files also for the same versions.
Added travis build setup for all of them conditioned by modifications done on the values files, the helm chart or tests.
Upgraded cluster to 1.17

### CODE REVIEW

File: helm/alfresco-content-services/6.0.N_values.yaml [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 20
+    timeoutSeconds: 10
+    livenessPercent: 400
+    livenessTransformPeriodSeconds: 600
+    maxTransforms: 10000
+    maxTransformSeconds: 1800
+
+# Declares the alfresco-tika service used by the content repository
```
Comment (gavincornwell) [inline helm/alfresco-content-services/6.0.N_values.yaml:222]: Looks like a copy/paste of the comment

File: helm/alfresco-content-services/6.0.N_values.yaml [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+# to transform office files
+transformmisc:
+  enabled: false
+  replicaCount: 2
```
Comment (gavincornwell) [inline helm/alfresco-content-services/6.0.N_values.yaml:226]: Do we need to specify all the values below if the "transformmisc.enabled" option is false?

File: helm/alfresco-content-services/6.1.N_values.yaml [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 20
+    timeoutSeconds: 10
+    livenessPercent: 400
+    livenessTransformPeriodSeconds: 600
+    maxTransforms: 10000
+    maxTransformSeconds: 1800
+
+# Declares the alfresco-tika service used by the content repository
```
Comment (gavincornwell) [inline helm/alfresco-content-services/6.1.N_values.yaml:222]: Same comments as 6.0.N file

File: helm/alfresco-content-services/values.yaml [hunk 4]
```diff
@@ -252,6 +253,7 @@ transformmisc:
     maxTransformSeconds: 1800
 
 aiTransformer:
+  enabled: true
```
Comment (gavincornwell) [inline helm/alfresco-content-services/values.yaml:257]: I don't see where this is being used?

**Review by gavincornwell — CHANGES_REQUESTED** [review]
One minor typo to address, other than LGTM.

File: helm/alfresco-content-services/values.yaml (hunk 4 above)
Comment (sergiuv2020) [reply helm/alfresco-content-services/values.yaml]: forgot it there, its not needed.

File: helm/alfresco-content-services/6.1.N_values.yaml (hunk 3 above)
Comment (sergiuv2020) [reply helm/alfresco-content-services/6.1.N_values.yaml]: done

File: helm/alfresco-content-services/6.0.N_values.yaml (hunk 2 above)
Comment (sergiuv2020) [reply helm/alfresco-content-services/6.0.N_values.yaml]: nope, removed them here and in 6.1

**Review by gavincornwell — APPROVED** [review]
LGTM, subject to testing.

**Pull request merged**